lib/src/fancy_on_boarding.dart

```diff
@@ -94,6 +96,7 @@ class _FancyOnBoardingState extends State<FancyOnBoarding>
           canDragRightToLeft: activeIndex < pageList.length - 1,
           slideUpdateStream: this.slideUpdateStream,
         ),
+        widget.showDoneButton ?
```

Comment:
There isn't need of showing an offset if the condition is false. Instead of using conditional operator, Please use 
```dart
if(widget.showDoneButton)
PositionedWidget()
```